guiEditor/src/diagram/guiGizmo.tsx

```diff
@@ -588,24 +648,53 @@ export class GuiGizmoComponent extends React.Component<IGuiGizmoProps, IGuiGizmo
                         return <img className="pivot-point" src={gizmoPivotIcon} style={style} key={index} />;
                     }
                     // compute which cursor icon to use on hover
-                    const delta = scalePoint.position.subtract(this.state.canvasBounds.center);
-                    const increment = Math.PI / 4;
-                    const angle = Math.atan2(delta.y, delta.x);
-                    let cursorIndex = Math.round((angle / increment) % 8);
-                    if (cursorIndex < 0) cursorIndex += 8;
-                    style.cursor = scalePointCursors[cursorIndex];
+                    const angleOfCursor = (defaultScalePointRotations[index]  + scalePoint.rotation);
+                    let angleAdjusted = angleOfCursor % (360);
+                    if (angleAdjusted < 0) angleAdjusted += 360;
+                    const increment = 45;
+                    let cursorIndex = Math.round(angleAdjusted / increment) % 8;
+                    const cursor = scalePointCursors[cursorIndex];
+                    const rotateClickAreaStyle = {
+                        top: 5 + 7 * scalePoint.verticalPosition,

```

Comment:
Can those values (5,7) be named constants so we can change them if and when needed?